test/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM teradatalabs/centos6-java8-oracle
+
+RUN \
+  yum install -y \
+    wget \
+    unzip \
+    hive \
+
+  # Setup CDH repo, pin the CDH distribution to a concrete version
+  && wget http://archive.cloudera.com/cdh5/one-click-install/redhat/6/x86_64/cloudera-cdh-5-0.x86_64.rpm \
+  && yum --nogpgcheck localinstall -y cloudera-cdh-5-0.x86_64.rpm \
+  && rm cloudera-cdh-5-0.x86_64.rpm \
+  && rpm --import http://archive.cloudera.com/cdh5/redhat/6/x86_64/cdh/RPM-GPG-KEY-cloudera \
+  && sed -i '/^baseurl=/c\baseurl=https://archive.cloudera.com/cdh5/redhat/6/x86_64/cdh/5.8.2/' /etc/yum.repos.d/cloudera-cdh5.repo \
+
+  # Install BATS bash test lib
+  && wget https://github.com/ArturGajowy/bats/archive/6959d91.zip -O /tmp/bats.zip \
```

Comment:
> why not to add this archive to this repo and keep as a source file?

Because it ensures the lib is accessible **somewhere**, and makes it clear where :) Otherwise we'd have to put a comment in there, and - as you know - all comments eventually lie

> it looks like bats is not used here yet, so maybe lets remove it for now?

yeah, it should come with the tests commit. Will fix that